scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+.navbar {
+    overflow: hidden;
+    background-color: white;
+    border-bottom: 1px solid black;
```

Comment:
use color variables

Reply:
this should be a shadow, to start set `box-shadow: 10px 10px 10px 10px $black` and start playing with the numbers until you find the one that suites you well.